mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -438,7 +438,6 @@ class LinearRegressionModel private[ml] (
     }
   }
 
-
```

Comment:
How to fix them? I've seen few commits with such a change so I thought this one would be enough to include this too. Please guide as I've got few other similar cases and don't know how to proceed with small but irritating cases.